common/package.json

```diff
@@ -32,6 +32,7 @@
     "koa-compose": "^4.1.0",
     "lazysizes": "^5.2.1",
     "lodash.debounce": "^4.0.8",
+    "lodash.flattendeep": "^4.4.0",
```

Comment:
Can you not use `Array.flat(Infinity)`?

Reply:
IE support, but also so can run tests as it's not in node